### DISCUSSION TITLE
If i were to fork the codebase for making dockerized django projects, instead of data science projects, is that ok?

### DISCUSSION THREAD

**Discussion by simkimsia** [kedro-org/kedro#1190] [Idea]
I'm studying cookiecutter, cruft, and kedro and i really like how kedro addresses some of the weaknesses of cookiecutter with prompts.yml but still build on top of cookiecutter.

If i were to fork the codebase and make a separate CLI for generating dockerized django projects out of cookiecutter templates, is that ok? I saw that the project is APACHE 2 license and as long as I don't use the kedro trademark name, I guess that's okay right?

**Comment by datajoely** [comment]
Hi @simkimsia you are absolute encouraged to do so! We're under an [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) licence so there is a small attribution requirement.

Other than that - the core team have spent a long time tinkering with the innards of both cookiecutter and Click so do shout if we can help :)

**Reply by simkimsia** [reply]
> a small attribution requirement.

Happy to do so. Erm how do i do this? i like to as above board and proper as I can. And i think i am okay to continue using as Apache 2 for my fork as well.

Do I just say inspired by Kedro in the README.md?

**Reply by datajoely** [reply]
Looks like the correct way is to include a NOTICE file https://opensource.stackexchange.com/a/8731
